XCUITests/FxScreenGraph.swift

```diff
@@ -335,71 +369,71 @@ extension Navigator {
         self.goto(URLBarOpen)
         let app = XCUIApplication()
         app.textFields["address"].typeText(urlString + "\r")
+        
         self.nowAt(BrowserTab)
     }
 
     // Opens a URL in a new tab.
     func openNewURL(urlString: String) {
-        self.goto(NewTabScreen)
+        self.goto(TabTray)
+        createNewTab()
         self.openURL(urlString: urlString)
     }
 
     // Closes all Tabs from the option in TabTrayMenu
     func closeAllTabs() {
-        self.goto(TabTrayMenu)
         let app = XCUIApplication()
-        app.collectionViews.cells["CloseAllTabsMenuItem"].tap()
-        self.nowAt(NewTabScreen)
+        app.buttons["TabTrayController.remoteTabsButton"].tap()
```

Comment:
Could be that there is a typo here, instead of 'remoteTabsButton' I think is 'removeTabsButton'

Reply:
Small thing...the comment above should be updated..

> // Closes all Tabs from the option in TabTrayMenu

Since TabTrayMenu is going to dissapear...in case you want to leave already updated in this PR